### PR TITLE
[REF-5638] Add builtin components to mlhub from mlpiper

### PR DESCRIPTION
**Pull request by sriram109**

### CODE REVIEW

**Review by sindhu-ghanta — APPROVED** [review]

File: components/Java/h2o_predictor/pom.xml [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <workingDirectory>${project.build.directory}</workingDirectory>
+                                    <executable>tar</executable>
+                                    <commandlineArgs>cvf ${project.name}.tar ${project.name}</commandlineArgs>
+                                </configuration>
+                            </execution>
+
```
Comment (yakov-g) [inline components/Java/h2o_predictor/pom.xml:146]: This is empty line :)

**Review by yakov-g — APPROVED** [review]

File: components/Python/Scikit-learn/model_connector_cloudera_cdsw/main.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
+from __future__ import print_function
```
Comment (liororama) [inline components/Python/Scikit-learn/model_connector_cloudera_cdsw/main.py:1]: Why model connector is under Sckit-learn?

File: components/Python/Scikit-learn/model_connector_h2o_open/component.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (liororama) [inline components/Python/Scikit-learn/model_connector_h2o_open/component.json:1]: H2o model connector under skit learn?

File: components/Python/Scikit-learn/model_connector_cloudera_cdsw/main.py (hunk 2 above)
Comment (PMjoydas) [reply components/Python/Scikit-learn/model_connector_cloudera_cdsw/main.py:1]: Please note, Model connector on CDSW is independent of Scikit-learn, please have it under components/Python/model_connector_cloudera_cdsw

Comment (PMjoydas) [reply components/Python/Scikit-learn/model_connector_cloudera_cdsw/main.py:1]: A kind request please have @PMjoydas included in the review for model-connectors and predictor being added for CDSW/H2o

**Review by harshilpm — APPROVED** [review]

**Pull request closed without merging**